cli/compose/loader/full-struct_test.go

```diff
@@ -313,10 +315,10 @@ func services(workingDir, homeDir string) []types.ServiceConfig {
 				{Target: "/var/lib/mysql", Type: "volume"},
 				{Source: "/opt/data", Target: "/var/lib/mysql", Type: "bind"},
 				{Source: workingDir, Target: "/code", Type: "bind"},
-				{Source: workingDir + "/static", Target: "/var/www/html", Type: "bind"},
+				{Source: filepath.Join(workingDir, "/static"), Target: "/var/www/html", Type: "bind"},
```

Comment:
remove the `/` from `/static` ? Also on opt below.

Reply:
yes the `/` is weird

Reply:
oups, I forgot to remove that one 🤦‍♂️